examples/jsm/loaders/LDrawLoader.js

```diff
@@ -2165,8 +2165,8 @@ class LDrawLoader extends Loader {
 		let code = null;
 
 		// Triangle and line colors
-		let fillColor = 0xFF00FF;
-		let edgeColor = 0xFF00FF;
+		let fillColor = '#FF00FF';
+		let edgeColor = '#FF00FF';
```

Comment:
So why was this change required? Will hex not work correctly.

Reply:
The color value is either the default given here, or a hex string parsed from the file. Rather than trying to choose setHex or setStyle conditionally, I'm ensuring the value is always a hex string.